.prettierrc.json

```diff
@@ -2,5 +2,7 @@
   "trailingComma": "es5",
   "tabWidth": 2,
   "semi": false,
-  "singleQuote": true
+  "singleQuote": true,
+  "bracketSpacing": true,
+  "proseWrap": "preserve"
```

Comment:
I was having some formatting issues when I saved so I added proseWrap preserve, let me know if this causes any issues.